core-bundle/contao/templates/twig/content_element/accordion.html.twig

```diff
@@ -0,0 +1,39 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{% block content %}
+    {% for element in elements %}
+        <h3 class="handorgel__header">
+            <button class="handorgel__header__button">{{ element.header }}</button>
+        </h3>
+        <div class="handorgel__content">
+            <div class="handorgel__content__inner">
+                {{ content_element(element.reference) }}
+            </div>
+        </div>
```

Comment:
Here is a suggestion that introduces some blocks and attributes, that allow doing the following in your individual template (or that of an extension):

- Add classes/attributes by defining variables like so:
   `{% set accordion_header_button = attrs().set('data-foo', 'bar') %}`
- Wrapping/changing individual blocks like so:
   `{% block element %}<section>{{ parent() }}</section>{% endblock %}`


```suggestion
        {% block element %}
            {% block element_header %}
                <h3{{ attrs(accordion_header|default).addClass('handorgel__header') }}>
                    <button{{ attrs(accordion_header_button|default).addClass('handorgel__header__button') }}>
                        {{- element.header -}}
                    </button>
                </h3>
            {% endblock %}
            {% block element_content %}
                <div{{ attrs(accordion_content|default).addClass('handorgel__content') }}>
                    <div{{ attrs(accordion_content_inner|default).addClass('handorgel__content__inner') }}>
                        {{ content_element(element.reference) }}
                    </div>
                </div>
            {% endblock %}
        {% endblock %}
```

Reply:
Can you update your suggestion to the current code? Otherwise I cannot commit it.

<img width="366" alt="" src="https://github.com/contao/contao/assets/1192057/4969ea15-9acb-421b-80d5-b76692c75c69">

Reply:
I added a new suggestions. It seems you cannot update the old one…